samples/csharp/end-to-end-apps/DeepLearning_ObjectDetection_Onnx/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <PropertyGroup>
+    <MicrosoftMLVersion>1.2.0</MicrosoftMLVersion>
```

Comment:
Is there a specific reason we are introducing an additional Directory.Build.props file here? It has the same updated version information found in the [Directory.Build.props in the broader samples folder](https://github.com/dotnet/machinelearning-samples/blob/master/samples/Directory.Build.props).

Reply:
Agree. If the version is the same than the rest of the samples it should use the file from the outer folder. A specific  Directory.Build.props file for a sample should only be used if the version being used is different than the version of the rest of the samples.

Reply:
Just deleted the file and tested. Versions resolved to those in top level folder.